piglet/src/test/java/org/apache/calcite/test/PigRelOpTest.java

```diff
@@ -139,6 +140,11 @@ private Fluent pig(String script) {
     return new Fluent(script);
   }
 
+  @BeforeClass
```

Comment:
Can we move the `@BeforeClass` methods into PigRelTestBase?

Reply:
I'd rather go for your alternative proposition